src/main/java/CompanyList.java

```diff
@@ -5,4 +5,14 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public boolean add(String companyName, int contactNumber, String contactEmail){
+        try{
+            Company newCompany = new Company(companyName, contactNumber, contactEmail);
+            companies.add(newCompany);
+            return true;
+        } catch(seedu.duke.InputMismatchException e){
```

Comment:
Perhaps add a space between the closing parenthesis and the opening curly braces.